### PR TITLE
FB - Update with typo3 and SPIP CMS

### PR DESCRIPTION
**Pull request by fabienOVH**
Wait for merging before SPIP and Typo3 guides PR are done

### CODE REVIEW

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 title: "Tutoriel - Installer manuellement un CMS sur mon hébergement"
 excerpt: "Découvrez comment installer manuellement un CMS sur votre hébergement"
-updated: 2024-03-20
+updated: 2024-03-25
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
updated: 2024-03-26
```

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 2]
```diff
@@ -89,6 +91,18 @@ Retrouvez plus d'informations sur notre page relative au [module PrestaShop](htt
 
 - Site officiel de [Grav](https://getgrav.org/){.external}
 
+#### Typo3
+
+**Typo3** est un CMS basé sur PHP, conçu pour le développement de sites web de toutes tailles, des petites aux grandes entreprises. Il s'appuie sur une base de données pour stocker le contenu et offre une vaste gamme d'extensions pour étendre ses fonctionnalités et personnaliser votre site web.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
**Typo3** est un CMS basé sur PHP, conçu pour le développement de sites web de toutes tailles, des petites aux grandes entreprises. Il s'appuie sur une base de données pour stocker le contenu. Il offre une vaste gamme d'extensions pour étendre ses fonctionnalités afin de personnaliser votre site web.
```

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 3]
```diff
@@ -89,6 +91,18 @@ Retrouvez plus d'informations sur notre page relative au [module PrestaShop](htt
...
+
+- Site officiel de [Typo3](https://typo3.fr/){.external}
+
+#### SPIP
+
+**SPIP** est un CMS conçu principalement pour la publication et la gestion de sites web éditoriaux comme des journaux ou des magazines en ligne. Basé sur PHP et s'appuyant sur une base de données SQL, il facilite la création de sites web enrichis de contenu textuel, graphique ou multimédia.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
**SPIP** est un CMS conçu principalement pour la publication et la gestion de sites web éditoriaux comme des journaux ou des magazines en ligne. Basé sur PHP et s'appuyant sur une base de données SQL, il facilite la création de sites web enrichis de contenu textuel, graphique et / ou multimédia.
```

**Comment by Kazarus4** [comment]
DO NOT MERGE WHILE SPIP AND TYPO3 PR's WILL BE READY TO MERGE TOO

**Review by OVH-Team-Guides — APPROVED** [review]

**Pull request merged**